core/inject.js

```diff
@@ -365,7 +365,9 @@ Blockly.createMainWorkspace_ = function(svg, options, blockDragSurface,
               console.log('WARNING: Moved object in bounds but there was no' +
                   ' event group. This may break undo.');
             }
-            Blockly.Events.setGroup(oldGroup);
+            if (oldGroup) {
```

Comment:
Setting oldGroup to something falsy is also reasonable, especially if that's the value it had before doing this.